src/bindings.ts

```diff
@@ -178,6 +178,7 @@ export enum BLST_ERROR {
   BLST_AGGR_TYPE_MISMATCH = 4,
   BLST_VERIFY_FAIL = 5,
   BLST_PK_IS_INFINITY = 6,
+  BLST_INVALID_SIZE = 7,
```

Comment:
Use a proper name like `EMPTY_AGGREGATE_ARRAY`, the numbers above are like this because that's what we got from the C lib. Move this error to below the comment `// Extra errors not in native bindings`